version.py

```diff
@@ -26,4 +26,4 @@
 #   historical commits sharing the same major version, in which case a
 #   minor version bump suffices.
 
-PROVISION_VERSION = '66.5'
+PROVISION_VERSION = '66.6'
```

Comment:
From the comment above: upgrading a dependency requires a major version bump, unless the upgraded dependency is backwards compatible with all of our historical commits sharing the same major version.

Reply:
Thanks, fixed!